ext/java/org/jruby/ext/stringio/StringIO.java

```diff
@@ -1319,6 +1319,14 @@ public IRubyObject string(ThreadContext context) {
         return string;
     }
 
+    @JRubyMethod(name = "to_s")
```

Comment:
i copied the original

```
    @JRubyMethod(name = "string")
    public IRubyObject string(ThreadContext context) {
        RubyString string = ptr.string;
        if (string == null) return context.nil;

        return string;
    }
```

maybe there is a better way, to use smth like

```java
@JRubyMethod(names= ...
```

Reply:
@headius - is there a shorter/better/dryier way to do this?

Reply:
I found an example: https://github.com/jruby/jruby/blob/f40ffbb554cae9f0d01729ccc413043669cfa393/core/src/main/java/org/jruby/javasupport/ext/JavaUtilRegex.java#L159

Reply:
The implementation of `string` and `to_s` are the same, right? You can just add another name to the `@JRubyMethod` annotation like `@JRubyMethod(name = {"string", "to_s"}`.